{"name": "test_login", "status": "failed", "statusDetails": {"message": "AssertionError: assert False", "trace": "self = <testCases.test_login.TestLogin object at 0x00000225FB670850>, setup = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"520d1d39cd6c315cb519562cc58479aa\")>\n\n    def test_login(self, setup):\n        #parameterization - pass (login_credentials - argument) to the test_login function for parameterization\n        #self.clinic = login_credentials[0]\n        #self.username = login_credentials[1]\n        #self.password = login_credentials[2]\n    \n        self.log.info(\"------------------------- Starting test login-----------------------------\")\n        self.log.info(\"---------------------opening the browser ----------------------------\")\n        self.driver = setup\n        self.lp = LoginPage.LoginPage(self.driver)\n        self.log.info(\"--------------Entering clinic -------------------------------\")\n        self.lp.set_clinic(self.clinic)\n        self.log.info(\"-------------entering username ------------------------\")\n        self.lp.set_username(self.username)\n        self.log.info(\"----------------entering password ---------------------\")\n        self.lp.set_password(self.password)\n        self.log.info(\"-------------Click on Login Button -----------------------\")\n        self.lp.click_login()\n        time.sleep(5)\n    \n        if self.lp.verify_valid_credentials():\n            self.log.info(\"---------Testcase Login Passed ---------------\")\n            self.log.info(\"-------------------Taking Screenshot ------------------\")\n            self.driver.save_screenshot(\".//Screenshots//login_test.PNG\")\n            assert True\n        else:\n            self.log.info(\"---------Testcase Login Failed ---------------\")\n            self.log.info(\"-------------Taking screenshot -------------------------\")\n            self.driver.save_screenshot(\".//Screenshots//login_test.PNG\")\n>           assert False\nE           assert False\n\ntestCases\\test_login.py:55: AssertionError"}, "attachments": [{"name": "log", "source": "fd8aa288-b469-41b9-9e25-62bf7cba5d53-attachment.txt", "type": "text/plain"}, {"name": "stdout", "source": "cbba58d5-d031-4dc8-b934-8a47401388f5-attachment.txt", "type": "text/plain"}], "start": 1692866290023, "stop": 1692866295636, "uuid": "3be0868e-4c3c-4527-9183-431d253ea930", "historyId": "75b0eab1774756fb370980a7afe61cb1", "testCaseId": "75b0eab1774756fb370980a7afe61cb1", "fullName": "testCases.test_login.TestLogin#test_login", "labels": [{"name": "parentSuite", "value": "testCases"}, {"name": "suite", "value": "test_login"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "DESKTOP-FU4KRG8"}, {"name": "thread", "value": "12500-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "testCases.test_login"}]}